src/components/Search.js

```diff
@@ -1,23 +1,35 @@
-import React from "react";
+import React, { useState } from "react";
+import SearchButton from "./SearchButton";
+
+const Search = ({ setSubmit }) => {
+  const [searchInput, setSearchInput] = useState("");
+
+  const handleSearchInput = e => setSearchInput(e.target.value);
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    setSubmit(searchInput);
+  };
 
-const Search = () => {
   return (
     <div className="search">
       <div className="page-header">
         <h4 className="text-left">Search Bookings</h4>
       </div>
       <div className="row search-wrapper">
         <div className="col">
-          <form className="form-group search-box">
+          <form className="form-group search-box" onClick={handleSubmit}>
```

Comment:
You should use the `onSubmit` for `form` elements instead of `onClick`. 

Although it will behave the same in this case, in other cases where you have a long form with many inputs,  `onSubmit` will be the event that is used to trigger validation of your input fields.